docs/components/clerk-provider.mdx

```diff
@@ -1,47 +1,35 @@
 ---
 title: '`<ClerkProvider>`'
-description: The <ClerkProvider> component wraps your React application to provide active session and user context to Clerk's hooks and other components.
+description: The <ClerkProvider> component provides session and user context to Clerk's hooks and components.
 ---
 
-The `<ClerkProvider>` component wraps your React application to provide active session and user context to Clerk's hooks and other components.
+The `<ClerkProvider>` component is required to integrate Clerk into your React application, providing session and user context to Clerk's hooks and components.
+
+The recommended approach is to wrap your entire app with `<ClerkProvider>` at the entry point to make authentication globally accessible. If you only need authentication for specific routes or pieces of your application, render `<ClerkProvider>` deeper in the component tree. This allows you to implement Clerk's functionality precisely where required without impacting the rest of your app.
 
 ## Usage
 
 The `<ClerkProvider>` component must be added to your React entrypoint.
 
 <Tabs items={["Next.js", "React"]}>
   <Tab>
-    <CodeBlockTabs options={["App Router", "Pages Router"]}>
-      ```tsx {{ filename: 'app/layout.tsx' }}
-      import React from 'react'
-      import { ClerkProvider } from '@clerk/nextjs'
-
-      export default function RootLayout({ children }: { children: React.ReactNode }) {
-        return (
+    ```tsx {{ filename: 'app/layout.tsx' }}
+    import React from 'react'
+    import { ClerkProvider } from '@clerk/nextjs'
+
+    export default function RootLayout({ children }: { children: React.ReactNode }) {
+      return (
+        <html lang="en">
+          <head>
+            <title>Next.js 13 with Clerk</title>
+          </head>
```

Comment:
is there a reason you added this? we don't have this anywhere else, certainly not the quickstart.